packages/flutter/test/cupertino/button_test.dart

```diff
@@ -7,56 +7,66 @@ import 'package:flutter/scheduler.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+const TextStyle testStyle = const TextStyle(
+  fontFamily: 'Ahem',
+  fontSize: 10.0,
+);
+
 void main() {
   testWidgets('Layout minimum size', (WidgetTester tester) async {
     await tester.pumpWidget(
-      new Center(child: new CupertinoButton(child: new Text(' '), onPressed: null))
+      new Center(child: new CupertinoButton(
+        child: new Text('X', style: testStyle),
+        onPressed: null,
+      ))
     );
     RenderBox buttonBox = tester.renderObject(find.byType(CupertinoButton));
     expect(
       buttonBox.size,
-      greaterThanOrEqualTo(const Size.square(48.0)),
-    );
-    expect(
-      buttonBox.size,
-      lessThan(const Size.square(100.0)),
+      // 1 10dp character + 16dp * 2 is smaller than the 48dp minimum.
```

Comment:
Does https://material.io/guidelines/layout/units-measurements.html#units-measurements-scaleable-pixels-sp apply for Ahem's font metrics?